src/components/Content/NewNote/NewNoteTitle/NewNoteTitle.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { NoteTitle } from './NewNoteTitle.styles';
+
+const NewNoteTitle = ({ setNoteInfo, title }) => {
+  const [noteTitle, setNoteTitle] = useState(title || '');
+
+  const handleTitleChange = (e) => {
+    setNoteTitle(e.target.value);
+    setNoteInfo((current) => ({ ...current, title: e.target.value }));
```

Comment:
Veo que en muchos eventos pasas el evento completo, no está mal si lo necesitas, pero creo que aquí podríamos destructurar sin problemas dado que solo necesitamos el value.
```suggestion
    const { value } = e.target;
    setNoteTitle(value);
    setNoteInfo((current) => ({ ...current, title: value }));
```